arangod/RocksDBEngine/RocksDBMethods.h

```diff
@@ -232,6 +232,29 @@ class RocksDBBatchedMethods final : public RocksDBMethods {
   rocksdb::WriteBatchWithIndex* _wb;
 };
 
+// INDEXING MAY ONLY BE DISABLED IN TOPLEVEL AQL TRANSACTIONS
+// THIS IS BECAUSE THESE TRANSACTIONS WILL EITHER READ FROM
+// OR (XOR) WRITE TO A COLLECTION. IF THIS PRECONDITION IS
+// VIOLATED THE DISABLED INDEXING WILL BREAK GET OPERARIONS.
+struct DisableIndexingOnRocksDBTrxGuard {
```

Comment:
can we use the existing `IndexingDisabler` struct for this? It does exactly the same, it only needs to be made publicly available to be used from here.